styles.css

```diff
@@ -0,0 +1,190 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    user-select: none;
+}
+body{
+    font-family: 'Epilogue', sans-serif;
+    font-size: 18px;
+    background-color:hsl(0, 0%, 98%);
+}
+.navbarContainer{
+    display: flex;
+    flex-direction: row;
+    margin: 1rem 2rem;
+    align-items: center;
+}
+.navbarContainer img{
+    display:none;
+}
+.navbarMenu{
+    display:flex;
+    flex-direction: row;
+    align-items: center;
+    width:100%;
+    justify-content: space-between;
+}
+.navbarItems{
+    list-style: none;
+}
+.navbarItems img{
+    vertical-align: middle;
+}
+.navbarItems li{
+    float:left;
```

Comment:
floata staramy się unikać i zamiast niego wykorzystywać flexbox - stało się to już w zasadzie standardem